bundles/org.openhab.binding.tellstick/src/main/resources/ESH-INF/thing/sensor.xml

```diff
@@ -87,59 +91,59 @@
 		<item-type>Number</item-type>
 		<label>Rainrate</label>
 		<description>The current rain rate</description>
-		<state pattern="%d" readOnly="true" />
+		<state pattern="%d mm" readOnly="true" />
 	</channel-type>
 
 	<channel-type id="raintotal">
 		<item-type>Number</item-type>
 		<label>Total Rain</label>
 		<description>Total rain</description>
-		<state pattern="%d" readOnly="true">
+		<state pattern="%d mm" readOnly="true">
```

Comment:
If you are going to use units I would suggest using UoM for this as well.

Reply:
Agreed, I will fix that too.